rpc_jobs/snapshot.yml

```diff
@@ -0,0 +1,95 @@
+- job:
+    name: RE35-Snapshot-Test
```

Comment:
Great -- I can see this job in action!  Are you actually looking to commit this test long-term though?

Reply:
Its been useful for testing, and may be useful for debugging and future testing.

Reply:
No objection to it going in, just wanted confirmation.